package.json

```diff
@@ -13,7 +13,8 @@
     "build-diagrams": "tools/diagrams.js",
     "build-toc": "tools/toc.js",
     "clean": "premove public resources static/_gen && hugo mod clean --all && hugo mod tidy",
-    "release": "np --no-publish && conventional-github-releaser -p angular"
+    "release": "np --no-publish && conventional-github-releaser -p angular",
+    "readme": "tools/readme.js"
```

Comment:
Are we missing this file ? or is it a leftover ?

Reply:
ugh... that generates the toc for the readme... that should be there, i'll add it back.

Reply:
it's back!